Formula/bartycrouch.rb

```diff
@@ -1,26 +1,13 @@
 class Bartycrouch < Formula
-  desc "Incrementally update your Strings files"
+  desc "Incrementally update/translate your Strings files"
   homepage "https://github.com/Flinesoft/BartyCrouch"
-  url "https://github.com/Flinesoft/BartyCrouch/archive/3.13.1.tar.gz"
-  sha256 "86a48d807e2028061cbdcd63ad021122cc60408327a8de32daf332344332bd92"
+  url "https://github.com/Flinesoft/BartyCrouch.git", :tag => "4.0.0", :revision => "3afdce4b875b6e8a573eaa30a225e709b7ee7b0a"
+  head "https://github.com/Flinesoft/BartyCrouch.git"
 
-  bottle do
```

Comment:
Please leave the `bottle do` block untouched, it will be handled automatically upon merging.

Reply:
Sorry, forgot this, fixed now!